README.md

```diff
@@ -1,2 +1,20 @@
 # hw01
 Repository for homework 1
+
+## Simon Yamawaki Shachter
+### Autobiography
+
+<img src="https://pbs.twimg.com/profile_images/575678523521683458/Psv53-eU_400x400.jpeg" width="200">
```

Comment:
Good work with html; ![ ](file path to image) would work, so long as your jpg is saved in the local repository and pushed back to Github as well.